Remove is deleted

henrikstranneheim opened the pull request:
## Description

### Added

-

### Changed

- Remove is deleted

### Fixed

-

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b remove-is_deleted -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

henrikstranneheim commented:
https://github.com/Clinical-Genomics/cg/pull/2790

ChrOertlin reviewed (approved):
Nice cleanup!

seallard commented on alembic/versions/2023_12_28_6c3da78d1af0_drop_is_deleted.py:
```diff
@@ -0,0 +1,25 @@
+"""drop_is_deleted
```
https://github.com/Clinical-Genomics/trailblazer/issues/325#issuecomment-1883219242

henrikstranneheim replied:
Cg PR merged

seallard reviewed (changes requested):
Looks good! We do call the endpoint from cg though, so we should create a separate PR which removes that call as well in cg.

henrikstranneheim commented on trailblazer/services/job_service.py:
```diff
@@ -1,6 +1,6 @@
 from datetime import datetime
-from trailblazer.constants import TrailblazerStatus
 
+from trailblazer.constants import TrailblazerStatus
```
@seallard I'm getting this diff due to imports not being sorted. Also, Ruff does not understand the init.py imports. Are you using the pre-commit tools?

henrikstranneheim commented:
### CLI and API 👍 
```
trailblazer scan
Error updating analysis for: case - setcolt : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC9963_slurm_ids.yaml'
Error updating analysis for: case - handyshrimp : [Errno 2] No such file or directory: '/home/proj/production/mutant/cases/handyshrimp/results/trailblazer_config.yaml'
Error updating analysis for: case - divinetomcat : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC5119_slurm_ids.yaml'
Error updating analysis for: case - HW2V2DRX2 : [Errno 2] No such file or directory: '/home/proj/production/flowcells/novaseq/runs/230504_A00689_0805_AHW2V2DRX2/slurm_job_ids.yaml'
Error updating analysis for: case - savingstag : [Errno 2] No such file or directory: '/home/proj/production/mutant/cases/savingstag/results/trailblazer_config.yaml'
Error updating analysis for: case - topewe : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC12566_slurm_ids.yaml'
Error updating analysis for: case - fullgoat : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC12567_slurm_ids.yaml'
Error updating analysis for: case - bolddrum : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC12696_slurm_ids.yaml'
Error updating analysis for: case - abovehusky : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC12791_slurm_ids.yaml'
Error updating analysis for: case - chiefgannet : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC12956_slurm_ids.yaml'
Error updating analysis for: case - grandoyster : [Errno 13] Permission denied: '/home/proj/production/microbial/results/reports/trailblazer/ACC12967_slurm_ids.yaml'
Status in Tower: coolkodiak - 282044 - WPjrjs23yxQwq
Error logging case - coolkodiak :  JSONDecodeError
slurm_load_jobs error: Invalid job id specified
Error updating analysis for: case - HK33MDRX3 : Command '['squeue', '--jobs', '5793325', '--states=all', '--format', '%A,%j,%T,%l,%M,%S']' returned non-zero exit status 1.
slurm_load_jobs error: Invalid job id specified
Error updating analysis for: case - HGYFNBCX2 : Command '['squeue', '--jobs', '5793259', '--states=all', '--format', '%A,%j,%T,%l,%M,%S']' returned non-zero exit status 1.
slurm_load_jobs error: Invalid job id specified
Error updating analysis for: case - HM2LNBCX2 : Command '['squeue', '--jobs', '5793260', '--states=all', '--format', '%A,%j,%T,%l,%M,%S']' returned non-zero exit status 1.
slurm_load_jobs error: Invalid job id specified
Error updating analysis for: case - HK33MDRX3 : Command '['squeue', '--jobs', '5793325', '--states=all', '--format', '%A,%j,%T,%l,%M,%S']' returned non-zero exit status 1.
Status in Tower: bossyeti - 282911 - 2VkkGmKqN3ODMw
Error logging case - bossyeti :  ValidationError
Status in Tower: bossmink - 282987 - 2bBeabAPHeAl0P
Updated status unitedbeagle - 282989: completed
Updated status largeturtle_rsync - 282990: completed
Updated status lightkodiak_rsync - 282991: completed
Updated status wholecivet_rsync - 282992: completed
Updated status upwardstork_rsync - 282993: completed
Updated status suitedgrub_rsync - 282994: completed
Updated status suitedgrub_rsync - 282995: completed
Updated status suitedgrub_rsync - 282996: completed
Updated status upwardstork_rsync - 282997: completed
Updated status wholecivet_rsync - 282998: completed
Updated status equalbug_rsync - 282999: failed
Updated status uphippo_rsync - 283000: completed
Updated status stableraven_rsync - 283001: completed
Updated status uphippo_rsync - 283002: completed
Updated status equalbug_rsync - 283003: failed
Updated status uphippo_rsync - 283004: completed
Updated status equalbug_rsync - 283005: failed
Updated status equalbug_rsync - 283006: failed
Updated status discretelocust_partial - 283007: failed
All analyses updated!
```
<img width="1217" alt="Screenshot 2024-01-11 at 09 47 10" src="https://github.com/Clinical-Genomics/trailblazer/assets/584738/528138a9-c246-4063-8680-de60e3737b71">

seallard reviewed (approved):
👍
